app/views/patients/update.js.erb

```diff
@@ -36,3 +36,14 @@ $('#change_log').html(
   // clear out fulfillment form
   $('#fulfillment').empty();
 <% end %>
+
+// flash success
+if ($( "#flash" ).length == 0) {
+  $('main > .container').prepend("<div id='flash'></div>")
+}
+
+$('#flash').removeClass().hide().fadeIn('slow')
+$('#flash').addClass("alert alert-success")
+$('#flash').html(
+  '<button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button><div id="flash_notice"> Patient data saved!</div>'
```

Comment:
Definitely don't feel dumb because of rails flash stuff! It is a purer example than most of programmers making it up as they go -- tough for me to think of an analog elsewhere in web dev.